roles/parse_operator_bundle/tasks/bundle_sanity_checks.yml

```diff
@@ -1,9 +1,10 @@
 ---
-# Note: we probably want to make this more robust in the future to make a sliding scale that can be adjusted on the fly based on configuration?
 - name: "Verify ocp versions are valid"
   fail:
-    msg: "com.redhat.openshift.versions must only contain v4.5"
-  when: (ocp_versions|length==0) or (ocp_versions|length>1) or ocp_versions[0] != "v4.5"
+    msg: "com.redhat.openshift.versions can not contain {{ item }}"
```

Comment:
should this say "The **label** com.redhat.openshift.versions can not contain {{ item }}"